sonar-python-plugin/src/test/java/org/sonar/plugins/python/IPynbProfileTest.java

```diff
@@ -38,5 +38,8 @@ public void profile() {
     assertThat(profile.rules()).extracting("repoKey").containsOnly("ipython");
```

Comment:
I see that "ipython" uses in many different places. Probably it is good to have constant for it? Like as in `CheckList.IPYTHON_REPOSITORY_KEY`

Reply:
Good point. I've updated some of the usages. In tests, I think it makes sense to use the literal for explicit expectations, though.